apps/api-extractor/src/extractor/Extractor.ts

```diff
@@ -158,6 +157,18 @@ export class Extractor {
     }
   }
 
+  /**
+   * Returns the normalized configuration object after defaults have been applied.
+   *
+   * @remarks
+   * This is a read-only object.  The caller should NOT modify any member of this object.
+   * It is provided for diagnostic purposes.  For example, a build script could write
+   * this object to a JSON file to report the final configuration options used by API Extractor.
+   */
+  public get config(): IExtractorConfig {
```

Comment:
Call this something other than `config`, because that implies that it's the same thing that got passed into the constructor.